sys/src/9/amd64/io.h

```diff
@@ -279,6 +279,51 @@ struct Pcisiz
 	int	bar;
 };
 
+/*
+ * DMG 06/11/2016 Add the definitions for generic PCI capability structure
+ * based on the VIRTIO spec 1.0. Below is the layout of a capability in the PCI
+ * device config space. The structure visible to the kernel repeats this, but
+ * adds fields to build the linked list of capabilities per device.
+ *
+ * struct virtio_pci_cap {
```

Comment:
I like // comments. But we need to get a style guide up that makes it clear they are OK. I suggest the coreboot style guide, which is the linux guide with // comments added as being ok.